index.bs

```diff
@@ -3423,6 +3423,34 @@ examples that violate the above rules are {{XMLHttpRequest}} and
 initialisms, even if they are repeated.
 </div>
 
+<h4 id="factory-naming" class="no-num no-toc">Start factory method names with `create` or `from`</h4>
```

Comment:
Is it a good idea for this to live under naming? What if we want to add guidance about what object these methods should live on?